packages/ssx-sdk/src/modules/Encryption.ts

```diff
@@ -88,13 +114,9 @@ class SignatureEncryption implements IEncryption {
     return key;
   };
 
-  public encrypt = async (data: Blob) => {
-    // blob => jwe
-    // get data blob as binary and structure as Uint8Array
-    // TODO: update approach to encode entire blob (including type) as Uint8Array
-    const binaryData = await data
-      .arrayBuffer()
-      .then(arraybuffer => new Uint8Array(arraybuffer));
+  public encrypt = async (data: any) => {
+    // json => jwe
```

Comment:
Instead of assuming data is a File blob, I make the assumption that it is JSON object. We can add support for blobs later (encryptBlob)